rpm/giflib.spec

```diff
@@ -1,88 +1,45 @@
 Summary: Library for manipulating GIF format image files
 Name: giflib
-Version: 4.2.3
+Version: 5.2.1
 Release: 1
 License: MIT
 URL: http://sourceforge.net/projects/giflib/
-Source0: http://downloads.sourceforge.net/giflib/giflib-%{version}.tar.bz2
-Group: System/Libraries
-
-Obsoletes: libungif <= %{version}-%{release}
-Provides: libungif <= %{version}-%{release}
+Source0: giflib-%{version}.tar.bz2
+Patch1: 0001-Disable-building-docs.patch
+Patch2: 0002-Avoid-timestamps.patch
 
 %description
 The giflib package contains a shared library of functions for
-loading and saving GIF format image files.  It is API and ABI compatible
-with libungif, the library which supported uncompressed GIFs while the
-Unisys LZW patent was in effect.
-
-Install the giflib package if you need to write programs that use GIF files.
-You should also install the giflib-utils package if you need some simple
-utilities to manipulate GIFs.
+loading and saving GIF format image files.
 
 %package devel
 Summary: Development tools for programs which will use the libungif library
-Group: Development/Libraries
 Requires: %{name} = %{version}-%{release}
-Provides: libungif-devel <= %{version}-%{release}
-Obsoletes: libungif-devel <= %{version}-%{release}
 
 %description devel
 This package contains the static libraries, header files and
 documentation necessary for development of programs that will use the
 giflib library to load and save GIF format image files.
 
-You should install this package if you need to develop programs which
-will use giflib library functions.  You'll also need to install the
-giflib package.
-
 %package utils
 Summary: Programs for manipulating GIF format image files
-Group: Applications/Multimedia
 Requires: %{name} = %{version}-%{release}
-Obsoletes: libungif-progs <= %{version}-%{release}
 
 %description utils
 The giflib-utils package contains various programs for manipulating
 GIF format image files.
 
-Install this package if you need to manipulate GIF format image files.
-You'll also need to install the giflib package.
-
-%package doc
-Summary:   Documentation for %{name}
-Group:     Documentation
-Requires:  %{name} = %{version}-%{release}
-
-%description doc
-%{summary}.
-
 %prep
-%setup -q -n %{name}-%{version}/%{name}
-%{__sed} -i 's/\r//' doc/lzgif.txt
+%autosetup -p1 -n %{name}-%{version}/%{name}
 
 %build
-%configure
-cd lib ; make %{?_smp_mflags} ; cd ..
-cd util ; make %{?_smp_mflags} ; cd ..
-
-MAJOR=`echo '%{version}' | sed 's/\([0-9]\+\)\..*/\1/'`
-%{__cc} $RPM_OPT_FLAGS -shared -Wl,-soname,libungif.so.$MAJOR -Llib/.libs -lgif -o libungif.so.%{version}
+%make_build
 
 %install
-rm -rf ${RPM_BUILD_ROOT}
-
-%make_install
-
-install -m 0755 -p libungif.so.%{version} $RPM_BUILD_ROOT%{_libdir}
-ln -sf libungif.so.%{version} ${RPM_BUILD_ROOT}%{_libdir}/libungif.so.4
-ln -sf libungif.so.4 ${RPM_BUILD_ROOT}%{_libdir}/libungif.so
+make PREFIX="%{_prefix}" LIBDIR="%{_libdir}" DESTDIR="%{?buildroot}" install
```

Comment:
Is there any reason to use `make install` instead of `%make_install` like before?

Reply:
Yes, the installation requires prefix and libdir etc and those don't work with the macro. Why it doesn't work for us but works for Fedora I have no idea, the macro seems identical and all.

The upstream dropped the autotools and replacement is plain Makefile. Commonly the prefix and libdirs are passed to configure/meson calls, but there is no such no.